front/src/components/ArticleDetailImageViewSlider.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * @author kouz95
+ */
+
+import React from "react";
+import ImageBox from "./ImageBox";
+import { ImageURISource, StyleSheet } from "react-native";
+import ImageViewSliderDot from "./ImageViewSliderDot";
+import Swiper from "react-native-swiper";
+import ActiveDot from "./ActiveDot";
+
+interface ImageSliderProps {
+  images: ImageURISource[];
+}
+
+export default function ArticleDetailImageViewSlider({
+  images,
+}: ImageSliderProps) {
+  let imageId = 0;
+
+  return (
+    <Swiper
+      loadMinimal={true}
+      loop={false}
+      dot={<ImageViewSliderDot />}
+      activeDot={<ActiveDot />}
+      paginationStyle={styles.pagination}
+      centerContent={true}
+      style={styles.container}
+    >
+      {images.map((imageURISource) => (
+        <ImageBox imageURI={imageURISource} key={++imageId} marginBottom={50} />
+      ))}
```

Comment:
실제 데이터로 받는 id로 사용할 예정이라면 key 값을 그대로 뒀다가 변경해도 되지만,
그럴 예정이 아니라면 map에서 제공하는 index로 key를 사용하는건 어떨까요?
map은 두번째 인자로 index를 옵셔널로 제공한답니다.

```suggestion
      {images.map((imageURISource, index) => (
        <ImageBox imageURI={imageURISource} key={index} marginBottom={50} />
      ))}
```